templates/base.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">

```

Comment:
using 2 whitespaces is the best practice
https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD
please, change everywhere